### PR TITLE
Update user-activity-report.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6098
Added info

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/user-activity-report.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ You must be a Teams service admin to make these changes. See [Use Teams administ
...
... period.</li><li>**1:1 calls** is the number of 1:1 calls that the user participated in during the specified time period.</li><li>**Audio time** is the total audio time that the user participated in during the specified time period. (Measured in minutes). </li><li>**Video time** is the total video time that the user participated in during the specified time period. (Measured in minutes). </li><li>**Screen sharing time** is the total screen share time that the user participated in during the specified time period. (Measured in minutes). </li>  <li>**Last activity** is the last date (UTC) that the user participated in a Teams activity.</li><li>**Other activity** tracks when the user is considered active but has a value of zero for chat messages, 1:1 calls, channel messages, total meetings, and meetings organized. Examples actions are when a user opens a channel message post but doesn't reply to it or when a user receives a private message and reads it but doesn't respond to it.</li> </ul>Note that **Group calls** has been replaced by **Meetings organized adhoc** and **Meetings participated adhoc**, in which the sum of these two values is equal to what was measured by **Group calls**.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/user-activity-report.md:43]:
```suggestion
|**5**   |The table gives you a breakdown of usage by user.   <ul><li>**Username** is the display name of the user. You can click the display name to go to the user's settings page in the Microsoft Teams admin center.</li><li>**Channel messages** is the number of unique messages that the user posted in a team chat during the specified time period.</li><li>**Reply messages** is the number of unique reply messages that the user posted in a team channel during the specified time period.</li> <li>**Post messages** is the number of unique post messages that the user posted in a team channel during the specified time period.</li><li>**Chat messages** is the number of unique messages that the user posted in a private chat during the specified time period.</li><li>**Urgent messages** is the number of urgent messages that the user posted in a  chat during the specified time period.</li><li>**Total meetings** is the total number of scheduled and ad hoc meetings a user participated in during the specified time period.</li><li>**Meetings organized** is the number of scheduled and ad hoc meetings a user organized during the specified time period.</li><li>**Meetings organized scheduled** is the number of scheduled meetings a user organized during the specified time period.</li><li>**Meetings organized ad hoc** is the number of ad hoc meetings a user organized during the specified time period.</li><li>**Meetings participated** is the number of the scheduled and the ad hoc meetings a user participated in during the specified time period.</li><li>**Meetings participated scheduled** is the number of the scheduled meetings a user participated in during the specified time period.</li><li>**Meetings participated ad hoc** is the number of ad hoc meetings a user participated in during the specified time period.</li><li>**1:1 calls** is the number of 1:1 calls that the user participated in during the specified time period.</li><li>**Audio time** is the total audio time that the user participated in during the specified time period (measured in minutes). </li><li>**Video time** is the total video time that the user participated in during the specified time period (measured in minutes). </li><li>**Screen sharing time** is the total screen share time that the user participated in during the specified time period (measured in minutes). </li>  <li>**Last activity** is the last date (UTC) that the user participated in a Teams activity.</li><li>**Other activity** tracks when the user is considered active but has a value of zero for chat messages, 1:1 calls, channel messages, total meetings, and meetings organized. Examples of actions are when a user opens a channel message post but doesn't reply to it or when a user receives a private message and reads it but doesn't respond to it.</li> </ul>Note that **Group calls** has been replaced by **Meetings organized ad hoc** and **Meetings participated ad hoc**, in which the sum of these two values is equal to what was measured by **Group calls**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**